akka-persistence/src/main/resources/reference.conf

```diff
@@ -41,6 +41,12 @@ akka.persistence {
     no-snapshot-store {
       class = "akka.persistence.snapshot.NoSnapshotStore"
     }
+    recovery {
+      # There is more time in between individual events gotten from the journal
+      # during replay than this the replay will be failed and behave which causes
+      # the actor to stop (after onRecoveryFailure has been invoked)
+      event-timeout = 30s
+    }
```

Comment:
I thought the event thing was better since the timeout is between individual events. I like `recovery-event-timeout`

With the "journal and snapshots" I meant, the idea to put it on `journal-plugin-fallback` sounds good, but then it also covers the snapshot so it also sort of belong to both. I'll just put it on journal-plugin-fallback but note that it also matters for reading the snapshot.